src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/SortByTest.java

```diff
@@ -30,7 +30,44 @@ public void testGetMappedFeature() {
                         "wfs?request=GetFeature&version=2.0&outputFormat=gml32&typename=gsml:MappedFeature&sortBy=gml:name");
         LOGGER.info("WFS GetFeature response:\n" + prettyString(doc));
 
-        // check order
+        checkSortByNameResponse(doc);
+    }
+
+    @Test
+    public void testGetMappedFeatureMultipleSortby() throws Exception {
+        String query =
+                "<wfs:GetFeature service=\"WFS\" version=\"2.0.0\"\n"
+                        + "    xmlns:gsml=\"urn:cgi:xmlns:CGI:GeoSciML-Core:3.0.0\"\n"
+                        + "    xmlns:wfs=\"http://www.opengis.net/wfs/2.0\"\n"
+                        + "    xmlns:fes=\"http://www.opengis.net/fes/2.0\"\n"
+                        + "    xmlns:gml=\"http://www.opengis.net/gml/3.2\">\n"
+                        + "    <wfs:Query typeNames=\"gsml:MappedFeature\" srsName=\"EPSG:4326\">\n"
+                        + "        <fes:Filter>\n"
+                        + "            <fes:PropertyIsLike wildCard=\"*\"\n"
+                        + "                singleChar=\".\" escapeChar=\"\\\">\n"
+                        + "                <fes:ValueReference>gsml:MappedFeature/gsml:specification/gsml:GeologicUnit/gml:description\n"
+                        + "                </fes:ValueReference>\n"
+                        + "                <fes:Literal>*</fes:Literal>\n"
+                        + "            </fes:PropertyIsLike>\n"
+                        + "        </fes:Filter>\n"
+                        + "        <fes:SortBy>\n"
+                        + "            <fes:SortProperty>\n"
+                        + "                <fes:ValueReference>gsml:MappedFeature/gml:name</fes:ValueReference>\n"
+                        + "                <fes:SortOrder>ASC</fes:SortOrder>\n"
+                        + "            </fes:SortProperty>\n"
+                        + "            <fes:SortProperty>\n"
+                        + "                <fes:ValueReference>gsml:MappedFeature</fes:ValueReference>\n"
```

Comment:
What property will be targeted here? The mapped feature ID?

Reply:
gsml:MappedFeature/gml:name